src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -19,7 +20,28 @@ const buildIncludeUrl = (href, type) => {
 
   const withTrailingHref = href.startsWith('/') ? href : `/${href}`;
 
-  return `${process.env.SIMORGH_INCLUDES_BASE_URL}${withTrailingHref}${resolvers[type]}`;
+  const includeUrl = `${process.env.SIMORGH_INCLUDES_BASE_URL}${withTrailingHref}${resolvers[type]}`;
+
+  const currentRendererEnv = () => {
```

Comment:
We already have the [url-parse](https://www.npmjs.com/package/url-parse) package installed, this allows you to grab the query strings off of a url, it might be safe/easier to use this? you may be able to replace some of this custom logic

Reply:
Ooh that's cool, this will make it a lot clearer as well. Thanks Chris!